src/admin-views/settings/tickets-commerce/stripe/connect/active/button.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * The Template for displaying the Tickets Commerce Stripe connection details.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ *
+ * @var string                                        $plugin_url      [Global] The plugin URL.
+ * @var TEC\Tickets\Commerce\Gateways\Stripe\Signup   $signup          [Global] The Signup class.
+ * @var TEC\Tickets\Commerce\Gateways\Stripe\Merchant $merchant        [Global] The Signup class.
+ * @var array                                         $merchant_status [Global] Merchant Status data.
+ */
+
+if ( false === $merchant_status['connected'] ) {
+	return;
+}
+
+?>
+<div class="tec-tickets__admin-settings-tickets-commerce-gateway-connected-row">
+	<div class="tec-tickets__admin-settings-tickets-commerce-gateway-connected-col1"></div>
+	<div class="tec-tickets__admin-settings-tickets-commerce-gateway-connected-col2">
+		<div class="tec-tickets__admin-settings-tickets-commerce-gateways-item-button">
+			
+		</div>
+		<a
+			class="tec-tickets__admin-settings-tickets-commerce-gateways-item-button-link"
+			href="<?php echo esc_url( 'https://dashboard.stripe.com/' ); ?>"
+			target="_blank"
```

Comment:
For security reasons, if we use `target="_blank"` we also need to add `rel="noopener noreferrer"`